From: SeongJae Park <sj@kernel.org>
Date: Sat, 12 Jul 2025 12:29:44 -0700
Subject: [PATCH] MAINTAINERS: rename DAMON section

DAMON section name is 'DATA ACCESS MONITOR', but DAMON is not only for
access monitoring but also for access-aware system operations.  Rename
it to simply DAMON.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 390829ae9803..30b6334d4513 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -6738,7 +6738,7 @@ S:	Maintained
 W:	https://docs.dasharo.com/
 F:	drivers/platform/x86/dasharo-acpi.c
 
-DATA ACCESS MONITOR
+DAMON
 M:	SeongJae Park <sj@kernel.org>
 L:	damon@lists.linux.dev
 L:	linux-mm@kvack.org
-- 
2.39.5

